src/site/_includes/spaces.njk

```diff
@@ -0,0 +1,158 @@
+---
+layout: default-next
+---
+<div class="landing-page spaces">
+
+  <header>
+    <div class="hero wrapper">
+      <div class="hero__columns switcher">
+        <div>
+          <div class="flow">
+              <h1 class="text-size-6">{{ title }}</h1>
+              <p>{{ description }}</p>
+          </div>
+        </div>
+        <div class="flow">
+          {% Img src="image/jxu1OdD7LKOGIDU7jURMpSH2lyK2/Rm7d6KWe3gabmU9IqZeq.png", alt="Globe", width="800", height="650" %}
+        </div>
+      </div>
+    </div>
+  </header>
+
+{% set spaces = collections['twitter-space'] %}
+<div class="spaces wrapper region all-center">
+  <h2 class="text-size-4  gap-bottom-size-2">{{ 'i18n.spaces.upcoming_spaces' | i18n(locale) }}</h2>
+  <div class="auto-grid all-center">
+
+  {% set upcomingSpaces = spaces | livePosts | filterInUpcoming('event_date') %}
+  {% set pastSpaces = spaces | livePosts | filterOutUpcoming('event_date') %}
+
+  {% if not upcomingSpaces.length %}
+  <p>A friendly message reminding to check for new upcomig spaces soon</p>
```

Comment:
Do you want to put in a **TODO: REPLACE_ME** or something prominent so that you don't accidentally launch with this?

Reply:
No need, I expect to update it today once i get the copy.